dist/formio.js

```diff
@@ -39,10 +39,10 @@ require('whatwg-fetch');
 var EventEmitter = require('eventemitter2').EventEmitter2;
 var copy = require('shallow-copy');
 
-/**
- * The Formio interface class.
- *
- *   let formio = new Formio('https://examples.form.io/example');
```

Comment:
Sorry, I did gulp build before checking in the code changes.    I can't see any difference in the comments but somehow git does....   Anyway,    The main two files that contain the code changes are src/components/selectboxes/SelectBoxes.js and src/components/select/Select.js.    The other 8 files are the result of gulp build.    I'll revert them and then re-submit.